oggm/graphics.py

```diff
@@ -249,8 +249,14 @@ def plot_domain(gdirs, ax=None, smap=None):
             poly_pix = geom['polygon_pix']
             smap.set_geometry(poly_pix, crs=crs, fc='white',
                               alpha=0.3, zorder=2, linewidth=.2)
-            for l in poly_pix.interiors:
-                smap.set_geometry(l, crs=crs, color='black', linewidth=0.5)
+            if isinstance(poly_pix, shpg.MultiPolygon):
+                for _poly in poly_pix:
+                    for l in _poly.interiors:
+                        smap.set_geometry(l, crs=crs, color='black',
+                                          linewidth=0.5)
+            else:
+                for l in poly_pix.interiors:
+                    smap.set_geometry(l, crs=crs, color='black', linewidth=0.5)
```

Comment:
Would the following work? 

```python
poly_pix = utils.tolist(poly_pix)
for _poly in poly_pix:
    for l in _poly.interiors:
        smap.set_geometry(l, crs=crs, color='black',
                                         linewidth=0.5)
```

Alternatively:

```python
if not isinstance(poly_pix, shpg.MultiPolygon):
    poly_pix = [poly_pix]
```

(just to avoid duplication)

Reply:
yes, I like that